instancetypes/cx/1/cx1.yaml

```diff
@@ -15,6 +15,15 @@ metadata:
       the IO threading from cores dedicated to the workload.
       In addition, in this series, the NUMA topology of the used
       cores is provided to the VM.
+
+      Number of spec.cpu.maxSockets is set to the same number as of 
+      guest cpus. This is a preventative measure. It avoids 
+      KubeVirt's default behavior of multiplying spec.cpu.guest for 
+      hotplugging, which can exceed KVM's supported CPU count and cause the VM to fail
```

Comment:
So I get why we need to do this as a workaround but is there a kubevirt/kubevirt bug looking for a better solution? Taking limits into account and setting MaxSockets to that value instead dynamically etc?

Reply:
Sounds like a good idea to derive the MaxSockets value from the limits.